module/applications/item/ability-use-dialog.mjs

```diff
@@ -470,8 +476,10 @@ export default class AbilityUseDialog extends Dialog {
    * @param {Event} event  Triggering change event.
    */
   _onChangeSlotLevel(event) {
-    const level = parseInt(event.target.value.replace("spell", ""));
-    const item = this.item.clone({ "system.level": level });
+    const level = event.target.value === "pact"
```

Comment:
Seems I commented this on the commit rather than the PR, so putting it here as well for consideration.

`this.item.actor.system.spells[event.target.value].level` *should* presumably work just fine here.